ministryofjustice/laa-review-criminal-legal-aid
# [CRIMAPP-911] Alter dependants grouping/banding

## Description of change
Alters dependants output age grouping/banding to reflect child's age at NEXT birthday

## Link to relevant ticket

https://dsdmoj.atlassian.net/browse/CRIMAPP-911
## Notes for reviewer
N/A
## Screenshots of changes (if applicable)
With the following ages:

```
1 year old
7 years old
8 years old
12 years old
13 years old
17 years old
```
### Before changes:


<img width="810" alt="Screenshot 2024-06-21 at 03 08 56" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/47113046/72d59bed-6be1-40b3-a962-0fe80ead072e">


### After changes:
<img width="785" alt="Screenshot 2024-06-21 at 03 08 31" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/47113046/c8eba409-adf1-46cc-b695-dcf510d582bf">


## How to manually test the feature
Add lots of dependants in Apply, note down the ages entered.

Submit application, check in Review
